internal/pkg/call/call_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package call
+
+import (
+	"fmt"
+	"go/ast"
+	"go/importer"
+	"go/parser"
+	"go/token"
+	"go/types"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"golang.org/x/tools/go/ssa"
+	"golang.org/x/tools/go/ssa/ssautil"
+)
+
+type fakeReferrer struct {
+	hasPathTo bool
+}
+
+func (f fakeReferrer) RefersTo(node ssa.Node) bool { return f.hasPathTo }
+
+func TestRegularCallReferredBy(t *testing.T) {
+	source := readFromTestData("test.go")
+	cases := []struct {
+		r              Referrer
+		parentFuncName string
+		want           bool
+	}{
+		{fakeReferrer{true}, "CallNoArgs", false},
+		{fakeReferrer{true}, "CallOneArg", true},
+		{fakeReferrer{false}, "CallOneArg", false},
+	}
+	for _, c := range cases {
+		ssaCall := getCall(source, c.parentFuncName)
+		call := Regular(ssaCall)
+		got := call.ReferredBy(c.r)
+		if got != c.want {
+			t.Errorf("call.ReferredBy(%v) == %v, want %v", call, got, c.want)
+		}
+	}
+}
+
+func readFromTestData(filename string) string {
```

Comment:
Typically we pass the `t` to all helper methods and:
1. Mark methods as helpers:  `t.Helper()`
2. Instead of `panic` call t.Fatal(err)